ddl/db_test.go

```diff
@@ -111,8 +111,6 @@ func setUpSuite(s *testDBSuite, c *C) {
 	_, err = s.s.Execute(context.Background(), "create database test_db")
 	c.Assert(err, IsNil)
 	s.s.Execute(context.Background(), "set @@global.tidb_max_delta_schema_count= 4096")
-
-	s.tk = testkit.NewTestKit(c, s.store)
```

Comment:
I mean we can remove the member `tk` from TestSuite.

Reply:
Got it, I'll try to fix it!

Reply:
@tangenta updated in https://github.com/pingcap/tidb/pull/18003/commits/56f9771ff8ee40416186b10b90fe406deb1fe10c https://github.com/pingcap/tidb/pull/18003/commits/2953b5874c01d756e7dae7fa854b429ad1d8d74c

I removed member `tk`, replaced usage of `s.tk` to `tk`, and added parameter `tk *testkit.TestKit` to `func (s *testDBSuite) mustExec` and `func (s *testDBSuite) mustQuery`, updated their usage accordingly.

PTAL, thanks!